### ISSUE TITLE
[BUG] transitionDuration in CSSParsedDecleration is parsing incorrectly

### ISSUE THREAD

**Issue by DanielBelmes** [niklasvh/html2canvas#2655]
Please make sure you are testing with the latest [release of html2canvas](https://github.com/niklasvh/html2canvas/releases). 
Old versions are not supported and issues reported for them will be closed.

# Please follow the general troubleshooting steps first:

- [x] You are using the latest [version](https://github.com/niklasvh/html2canvas/releases)
- [x] You are testing using the non-minified version of html2canvas and checked any potential issues reported in the console

### Bug reports:

I'm exporting a canvas of Kibana and it is failing when parsing the style 
CSSParsedDeclaration.transitionDuration = "0.15s, 0.15s, 0.15s, 0.15s" Tokens parsed out too.

```
[
    {
        "type": 15,
        "number": 0.15,
        "flags": 8,
        "unit": "s"
    },
    {
        "type": 4
    },
    {
        "type": 31
    },
    {
        "type": 15,
        "number": 0.15,
        "flags": 8,
        "unit": "s"
    },
    {
        "type": 4
    },
    {
        "type": 31
    },
    {
        "type": 15,
        "number": 0.15,
        "flags": 8,
        "unit": "s"
    },
    {
        "type": 4
    },
    {
        "type": 31
    },
    {
        "type": 15,
        "number": 0.15,
        "flags": 8,
        "unit": "s"
    }
]
```

When it reaches parseComponentValue it triggers `throw new SyntaxError(`Error parsing CSS component value, multiple values found when expecting only one`);` Because of {type: 4}. 

PR that caused this https://github.com/niklasvh/html2canvas/pull/2632

### Specifications:

 * html2canvas version tested with: 1.3.0
 * Browser & version: Chrome 91.0.4472.114
 * Operating system: Mac Os Catalina

**Comment by AgentSmith0** [comment]
I can confirm this issue. I have the same problem.

**Comment by niklasvh** [comment]
Thanks for the report, should be fixed with 1.3.1
